modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi.adoc
+
+[id="csi-drivers-supported_{context}"]
+= CSI drivers supported by {product-title}
+
+{product-title} installs certain CSI drivers by default, giving users storage options that are not possible with in-tree volume plug-ins.
+
+To create CSI-provisioned persistent volumes that mount to these supported storage assets, {product-title} installs the necessary CSI driver Operator, the CSI driver, and the required storage class in the `openshift-cluster-csi-drivers` namespace by default.
+
+The following table describes the CSI drivers that are installed with {product-title} and which CSI features they support, such as volume snapshots, cloning, and resize. 
+
+.Supported CSI drivers and features in {product-title}
+[cols=",^v,^v,^v,^v", width="100%",options="header"]
+|===
+|CSI driver  |CSI volume snapshots  |CSI cloning  |CSI resize |CSI ephemeral volumes
+
+|AWS EBS (Tech Preview) | ✅ | ✅ | ✅ | ✅
+|OpenStack Cinder | ✅ | ✅ | ✅ | ✅
+|OpenStack Manila | ✅ | ✅ | ✅ | ✅
+|Red Hat Virtualization (oVirt) | ✅ | ✅ | ✅ | ✅
+|===
+
+[IMPORTANT]
+====
+If your CSI driver is not listed in the preceding table, you must follow the installation instructions provided by your CSI storage vendor to use their supported CSI features.
+====
```

Comment:
@duanwei33 and @qinpingli - Per Wei's suggestion, which I agree with, I have added a table that outlines the CSI drivers we currently ship with and support, along with the CSI features that are available for them. I do not know if this list is accurate or complete, so please comment.

Reply:
@bobfuru  Thanks for considering. I added comments inline for GCE missing and some unsupported features for other csi drivers. 
@jsafrane I'm not sure for the oVirt part, could you take a look for oVirt and have an overview review for all these csi drivers?